pkg/apicapi/apicapi.go

```diff
@@ -437,12 +450,18 @@ func (conn *ApicConnection) runConn(stopCh <-chan struct{}) {
 			syncHook <- fullSync{}
 		}()
 	}
-
+	minInterval := 90 * time.Second
 	refreshInterval := conn.RefreshInterval
-	if refreshInterval == 0 {
+	var diffInterval time.Duration
+	if refreshInterval <= minInterval {
 		refreshInterval = defaultConnectionRefresh
+		diffInterval = 0 * time.Second
+	} else {
+		refreshInterval = minInterval
+		diffInterval = 60 * time.Second
 	}
-	refreshTicker := time.NewTicker(refreshInterval)
+//	diffInterval := 60 * time.Second
+	refreshTicker := time.NewTicker(refreshInterval - diffInterval)
```

Comment:
@apoorva11029  You can absorb https://github.com/noironetworks/aci-containers/pull/292 into this PR